doc/source/webapp_ui/sublattices/sublattices_graph.rst

```diff
@@ -1,14 +1,22 @@
 ============================
 Sublattices Transport Graph
 ============================
+
+When you open the Graph View page for a lattice that contains sublattices, the sublattices are displayed as simple (unexpanded) nodes in the transport graph.
```

Comment:
Can we add a link to concepts page of sublattice here ? as well as give a small snippet of what a sublattice is just to reinforce the primitive ?

Reply:
Done.